src/o365/tenant/commands/report/tenant-report-servicesusercounts.spec.ts

```diff
@@ -0,0 +1,117 @@
+import commands from '../../commands';
```

Comment:
File renamed from `tenant-report-activeuserdetail.ts` to `report-activeuserdetail.ts` to match the naming conventions